Dockerfile

```diff
@@ -48,9 +48,11 @@ RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
     && apt-get update -qq \
     && DEBIAN_FRONTEND=noninteractive apt-get install -qq postgresql-client-12
 
-# Install Google Chrome for browser tests
-RUN curl -sSL https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o /tmp/chrome.deb \
-    && apt-get -y install --no-install-recommends /tmp/chrome.deb \
+# Install Google Chrome for browser tests with values {"chrome_version": "126.0.6478.182-1"}
+# Fixed following Odoo's Runbot guideline https://github.com/odoo/runbot/commit/f18a1adfc387321dac907735e8c3a77a15b2607f
+RUN curl -sSL https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_123.0.6312.58-1_amd64.deb -o /tmp/chrome.deb \
```

Comment:
Why not use version 126 like Odoo does?
Perhaps we should have a Dockerfile ARG with the chrome version so we can use recent versions for recent Odoo?

Reply:
Oh, sorry, I did so many tests I forgot to fix that back

Reply:
> Perhaps we should have a Dockerfile ARG with the chrome version so we can use recent versions for recent Odoo?

Yes, maybe, as some older versions won't be supported by some recent changes in the Chrome/Chromium support